skycoin/skywire
# AppStats: Connection Duration Addition to API

Did you run `make format && make check`? yes

Fixes #895 

 Changes:	
- Added Connection Duration to the vpn-client

TODO:
- [X] Adds the duration to the API when user requires visor overview / connections stats.
- [X] Also generates the mock and readmegen

How to test this PR:
- Run the `vpn-client` app from the visor via UI / cli
- Run the VPN UI
- Connect to any vpn-server
- Check the connection duration field in json

<img width="831" alt="Screen Shot 2021-10-04 at 21 43 02" src="https://user-images.githubusercontent.com/72057554/135957500-9db7ea8d-9f57-4925-a68f-7027c3984192.png">
